modules/constructor/constructor-front-pocket/constructor-front-pocket.services.js

```diff
@@ -0,0 +1,130 @@
+const { ObjectId } = require('mongoose').Types;
+const ConstructorFrontPocket = require('./constructor-front-pocket.model');
+const Currency = require('../../currency/currency.model');
+const { deleteFiles, uploadFiles } = require('../../upload/upload.service');
+const {
+  FRONT_POCKET_NOT_FOUND,
+  FRONT_POCKET_ALREADY_EXIST,
+  IMAGE_NOT_PROVIDED,
+} = require('../../../error-messages/constructor-front-pocket-messages');
+
+class ConstructorFrontPocketService {
+  async getAllConstructorFrontPocket({ skip, limit }) {
+    const items = await ConstructorFrontPocket.find().populate(
+      'material',
+    )
+      .skip(skip)
+      .limit(limit);
+
+    const count = await ConstructorFrontPocket.find().countDocuments();
+    return {
+      items,
+      count,
+    };
+  }
+
+  async getConstructorFrontPocketById(id) {
+    const foundConstructorFrontPocket = await ConstructorFrontPocket.findById(id).populate(
+      'material',
+    );
+    if (foundConstructorFrontPocket) {
+      return foundConstructorFrontPocket;
+    }
+    throw new Error(FRONT_POCKET_NOT_FOUND);
+  }
+
+  async calculatePrice(price) {
```

Comment:
Please move this method to separate currency file.

Reply:
Fixed